packages/suggestion/src/suggestion.ts

```diff
@@ -66,6 +68,20 @@ export interface SuggestionOptions<I = any, TSelected = any> {
    */
   decorationClass?: string
 
+  /**
+   * The content of the decoration node.
+   * @default null
+   * @example 'Type to search...'
+   */
+  decorationContent?: string
```

Comment:
So if there's no content, we don't create a decoration?
Maybe we could explain this in the JSDoc